pkg/testutils/mockmaps/lbmap.go

```diff
@@ -59,10 +59,6 @@ func (m *LBMockMap) UpsertService(p *datapathTypes.UpsertServiceParams) error {
 	if !found {
 		frontend := lb.NewL3n4AddrID(lb.NONE, p.IP, p.Port, p.Scope, lb.ID(p.ID))
 		svc = &lb.SVC{Frontend: *frontend}
-	} else {
-		if p.PrevBackendsCount != len(svc.Backends) {
-			return fmt.Errorf("Invalid backends count: %d vs %d", p.PrevBackendsCount, len(svc.Backends))
-		}
```

Comment:
Why is this mock maps code getting deleted? Is this intentional?

Reply:
@aditighag 
I deleted mock code intentionally because when backend count  changes by assigning backend array in service testcase,   test case fails.

Reply:
@aditighag 
Is there problem in deleting mock code?